packages/react/src/components/Chip/chip.scss

```diff
@@ -17,23 +17,99 @@
  */
 
 .oxygen-chip {
+  border-radius: var(--oxygen-customComponents-Chip-properties-border-radius);
+  border: var(--oxygen-customComponents-Chip-properties-border);
+  text-transform: var(--oxygen-customComponents-Chip-properties-text-transform);
+  font-weight:  var(--oxygen-customComponents-Chip-properties-font-weight);
+  line-height: var(--oxygen-customComponents-Chip-properties-line-height);
+
   &-premium {
-    background: var(--oxygen-customComponents-Chip-properties-premium-background);
-    color: var(--oxygen-customComponents-Chip-properties-premium-text-color);
+    background:  var(--oxygen-customComponents-Chip-properties-premium-background);
+    border: var(--oxygen-customComponents-Chip-properties-border);
+    border-radius: var(--oxygen-customComponents-Chip-properties-border-radius);
+    border-color: var(--oxygen-customComponents-Chip-properties-premium-border-color);
+    font-weight:  var(--oxygen-customComponents-Chip-properties-font-weight);
+    line-height: var(--oxygen-customComponents-Chip-properties-line-height);
+
+    .MuiChip-icon {
+      fill: var(--oxygen-customComponents-Chip-properties-premium-border-color);
+    }
+
+    .MuiChip-label {
+      -moz-background-clip: var(--oxygen-customComponents-Chip-properties--moz-background-clip);
```

Comment:
We need to check if we are using `autoprefixer` with the `postcss` plugin.
If so, we can remove these vendor prefixes.

Reply:
fixed with https://github.com/wso2/oxygen-ui/pull/222/commits/0a43bdaf0351654be4b2ded8b5dd031b7190fe22